underfs/hdfs/src/main/java/tachyon/underfs/hdfs/HdfsUnderFileSystem.java

```diff
@@ -121,14 +123,13 @@ public void close() throws IOException {
   @Override
   public FSDataOutputStream create(String path) throws IOException {
     IOException te = null;
-    int cnt = 0;
-    while (cnt < MAX_TRY) {
+    RetryPolicy retryPolicy = new CountingRetry(MAX_TRY);
+    while (retryPolicy.attemptRetry()) {
```

Comment:
That would just involve a change in the interface. Looks to me to just be more expressive to say attemptRetry though the meaning is implicit here. 
I am ok with the change but if it makes sense I can raise a separate JIRA for this change since it might involves changes in other places and preferably a separate pull req?